pkg/prowgen/prowgen_konflux.go

```diff
@@ -75,12 +81,6 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 						}
 						// For non-existent branches we keep going and use downstreamVersion for versionLabel.
 					} else {
-						soProjectYamlPath := filepath.Join(soRepo.RepositoryDirectory(),
```

Comment:
Sorry, too late. project.ReadMetadataFile needs to be done after running GitCheckout., and it won't be available if the branch doesn't exist. I'm not sure this change will work.

Reply:
@Kaustubh-pande Why did you move this up? I don't get it.

Reply:
I guess this was a leftover from https://github.com/openshift-knative/hack/pull/406#discussion_r1830927550. I'll submit a PR with the fix

Reply:
https://github.com/openshift-knative/hack/pull/409